spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/SharedServerContextConfiguration.java

```diff
@@ -119,6 +120,18 @@ public ZooKeeperConnection zooKeeperConnection() {
 		@Value("${zk.namespace:}")
 		protected String zkNamespace;
 
+		@Value("${zk.client.sessionTimeout:" + ZooKeeperConnection.DEFAULT_SESSION_TIMEOUT + "}")
+		protected int zkSessionTimeout;
+
+		@Value("${zk.client.connectionTimeout:"+ ZooKeeperConnection.DEFAULT_CONNECTION_TIMEOUT +"}")
+		protected int zkConnectionTimeout;
+
+		@Value("${zk.client.initialRetryWait:" + ZooKeeperConnection.DEFAULT_INITIAL_RETRY_WAIT + "}")
+		protected int zkRetryWait;
```

Comment:
here as well I guess: zkInitialRetryWait for the var name, to be consistent withe the others mapping to the properties from the yml file and the constant names themselves.